src/Aspire.Hosting/Dcp/DcpHostService.cs

```diff
@@ -70,9 +68,13 @@ public async Task StartAsync(CancellationToken cancellationToken = default)
         await _appExecutor.RunApplicationAsync(cancellationToken).ConfigureAwait(false);
     }
 
-    public Task StopAsync(CancellationToken cancellationToken = default)
+    public async Task StopAsync(CancellationToken cancellationToken = default)
     {
-        return Task.CompletedTask;
+        _shutdownCts.Cancel();
+        if (_logProcessorTask is { } task)
+        {
+            await task.ConfigureAwait(ConfigureAwaitOptions.SuppressThrowing);
```

Comment:
Can this be changed to ignore expected errors (I'm guessing `OperationCanceledException`) and log unexpected errors?

Reply:
Sure thing, updated